### ISSUE TITLE
Installer Installation Failed

### ISSUE THREAD

**Issue by neminovno** [BowlerHatLLC/feathers-sdk-manager#7]
![image](https://user-images.githubusercontent.com/1468887/34909414-a3c62d40-f8a0-11e7-9f4f-3faba70559b3.png)

Log:

> Feathers SDK Manager 1.2.0 windows
> Loading configuration file: http://feathersui.com/sdk/feathers-sdk-manager/sdk-manager-config-1.0.xml
> Loading the Feathers SDK configuration data failed. The file was not found on the server. [IOErrorEvent type="ioError" bubbles=false cancelable=false eventPhase=2 text="Error #2032" errorID=2032]

In browser, not found [file](http://feathersui.com/sdk/feathers-sdk-manager/sdk-manager-config-1.0.xml) opens OK.

This issue does not look like https://github.com/BowlerHatLLC/feathers-sdk/issues/49, at least visually.
I have latest AIR 28.0.0.127 installed on Win7.

I tried running it with:
`C:\Program Files (x86)\Feathers SDK Manager>"Feathers SDK Manager.exe" -config sdk-manager-config-1.0.xml`
but nothing happens:
![image](https://user-images.githubusercontent.com/1468887/34909567-0c7b2a68-f8a4-11e7-9dda-854e891261a8.png)

After repo download, create project, change link to https..., run - it works ok!

Created pull request https://github.com/BowlerHatLLC/feathers-sdk-manager/pull/6

**Comment by joshtynjala** [comment]
I wonder why I haven't personally encountered this issue before... I always test installing the SDK when I release a new version, and I made the switch to forced https on the server quite a while ago. What could it be about your computer that can't handle the redirect while mine had no complaints?

**Comment by neminovno** [comment]
Hm... I thought that redirect to https could be the reason.

I just remembered that recently I had similar issue. After downloading some app, its installer immediately stopped saying:

> An error occurred trying to download 'http://...'

but that URL worked in browser. After looking in its log in the end it says:

> URLDownloadToCacheFile failed with HRESULT '-2146697202'
> Error: An error occurred trying to download 'http://...'

Then I found [Error message when you try to install a ClickOnce application from a Web server: "An error occurred trying to download "](https://support.microsoft.com/en-us/help/934387/error-message-when-you-try-to-install-a-clickonce-application-from-a-w) but didn't try mentioned workaround.

Now I did, and after disabling _Warn if changing between secure and not secure mode_ option in IE 
![image](https://user-images.githubusercontent.com/1468887/34911551-fd994a92-f8cc-11e7-990c-e37f364fd62f.png)
Feathers SDK Manager started up without error!

Note: I never use IE and I thought I uninstalled it. Don't know how that option enabled.. Could be that IE was reinstalled during windows update or after installing Office 2013, and enabled that option.
